### DISCUSSION TITLE
Prisma Scalar Type Limit

### DISCUSSION THREAD

**Discussion by StevenTannn** [prisma/prisma#17910] [Q&A]
Hi guys, can I query limit scalar list query? Example model:

```
model Test {
  id                  String           @id @default(auto()) @map("_id") @db.ObjectId
  results             TestResults[]
}
type TestResults{
  batch   Int
}
```

Can I do a query that only get 1 results from this scalar list? Thanks in advance for helping

I expect the response will be one data from this array and not all

**Comment by ludralph** [comment]
Hi @StevenTannn 👋 

you can simply do 
```js
const res = await prisma.test.findMany({
  take: 1
})
````

**Reply by nikolasburk** [reply]
Maybe `findFirst` would also work in this case? That way no pagination arguments are needed?

```ts
const res = await prisma.test.findFirst()
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
